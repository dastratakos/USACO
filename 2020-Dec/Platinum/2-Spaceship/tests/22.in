60 57 60
101110111110100011111011011110011001010111111101101100111011
111111101111111001100011111110110111011111111111110011111101
110101110011010110100111110101010100111110011111101001010010
000110110111100111010011110101111101111011011111001101100100
111101110111011011100111110011101110111111101111110100111011
001100111011011110110111111101001111000111111110111001111011
001110101110111011101111101100110110100101111110110100111000
111011011111001110011111111111101010001001011111101101101011
101101111110111001011101110000111011111010111111110011010011
011100010101001111011111011110111011110101101111011111101011
110010101111011111111011000101001100011111111011101111111000
001011110101111111011110011101101111110111110100110010111110
011101101010011111100101001111001100111101111011101110001011
001101011101010010111111101110100010101100110011000001011110
111111011110111111111101111110100111111011010011101101111011
101011111000010111101100101111011110101101101101111110001011
111101110001111111010111000110011011101001101011110111111111
101110111100101100101110111001100011110010111111101011110001
101111101111001111001111011101011010011010110111111101101111
101110111110011010001101011110111101011110011110110001010001
110101111111111110110111110011111111101111100010011111001100
110010000111110111011101111011111110011101111011001111011001
011110111011101011111110110100111110011111110010101110110110
110101011010101110111101001111101110110101111101011100101101
010011011111011101111111110111101000111111111101110011011101
000111111111111101111111011110111000111111011011101111101001
111110111111001111111111111110010011111111110110110111010101
001110000111111001101111110111010111101111011111000110110101
111101101111111110110110101101011101011001011111011011100011
101011111000001110111011111111110111111011110001101110011111
110101110111110011011001000100110101100001111001111110010011
111111111001111011011110100110110110100111010111000001011011
110011010101100101010001011111111111011111001111100111101101
011001111010111111110110110011110010110111101111011111101100
011101011010010100110100101111111010111010111110110000011010
101111100110110001000101001110111111011010001011011010111010
110001010111111010110010111001111001100101010111111010111101
111111110011011111100101101000100001111101101010010111100001
011011011010111110110011110001101011111000111101111110100111
110110111110111011111000011000100111100011111011111101110010
010001011111011111100011101000111011111110110010111100111111
010110101111111011111111011101101110111011000000011110100111
011101101101111101111111111111110010111100101011011101001110
000101100011111100000011001101101011011101010001101001001101
101011110011101111101011101011110111101101111110110111010101
011010001001011001001011101110111111110011111011111101110110
110101111110010101000101001101101010000001111011101000110100
001010111010111111111110110101111111101011111111011011110000
110111010110101010101001111011111111000110101111111011111100
000001010011110110101111101001100111111101101111111000110111
111101110011111110111011111111011011111110110011111101000100
010001101011111101000010110111111110111101111100101111001111
010100111110111100101000111111100111111111101010001111110101
100111111011011111010011100111011011001011100011011111110111
011111101001000111111100001001011011011111011010010101011101
011111111101001110011110110100010111111111110110000011010011
011001111110101011101010101100110100111011111001111110011011
111011111010100011111101111111111110110101110110110001101100
111011011111101000110111001111101010111101001000000111110101
101011101011011101101101110111101110110111111111011010110100
5 16 4 2
5 27 2 46
3 45 1 22
3 21 4 38
3 31 3 23
45 37 2 19
39 54 2 16
3 36 17 28
2 1 3 12
1 23 34 60
11 55 2 47
35 34 4 39
5 34 1 7
21 58 57 51
4 1 3 10
1 12 3 30
2 26 2 10
2 10 5 20
1 12 3 2
1 30 4 54
5 53 22 3
2 7 46 32
2 13 3 9
4 18 4 17
4 22 37 24
5 18 3 54
38 46 1 33
2 44 15 27
14 14 1 46
4 52 19 57
1 7 4 2
5 39 4 21
1 37 2 49
1 39 5 25
1 9 53 2
27 57 4 44
3 60 6 6
5 16 2 24
6 47 41 25
4 41 50 51
1 42 3 10
45 37 2 9
3 10 39 51
1 48 3 16
3 30 2 44
5 26 1 36
5 5 1 10
27 13 4 11
1 9 2 25
1 59 7 6
2 25 4 20
2 14 1 35
41 13 5 56
55 13 34 30
2 30 5 32
5 60 5 32
4 9 2 54
3 19 5 43
4 48 2 48
55 36 55 36
