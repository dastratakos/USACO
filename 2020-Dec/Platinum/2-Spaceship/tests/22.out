525368864
125035055
487209096
717624418
978754211
110924884
682757405
148936859
384179531
378994459
177412412
407959229
705362641
865660932
639800528
470551355
282135713
749044837
775053703
998279264
823669429
549528956
974605313
614800147
296248403
814979663
743562292
505892908
484574039
523208340
305395093
628788311
86941239
951833870
930037117
121256317
202175204
3314000
365229205
73086769
801438014
586349344
573639778
777275908
60711619
953338763
227092256
229447437
846047672
357991646
378709973
820962447
938029219
953679183
537559899
949663547
222505121
424011669
109539133
782676211
